Three Eyed Raven/CharactersViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  CharactersViewController.swift
+//  Three Eyed Raven
+//
+//  Created by Fiona Thompson on 3/27/17.
+//
+//
+
+import UIKit
+
+class CharactersViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, UISearchBarDelegate, UITabBarDelegate{
+
+    @IBOutlet weak var tableView: UITableView!
+    
+    let searchBar = UISearchBar()
+    let tabBar = UITabBar()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.delegate = self
+        tableView.dataSource = self
+        searchBar.delegate = self
+        
+        tabBar.delegate = self
+        self.tabBar.barTintColor = UIColor.black
+        
+        self.navigationItem.titleView = searchBar
+
+
+        // Do any additional setup after loading the view.
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 1
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "CharacterCell") as! CharacterCell
+        return cell
+    }
+    public func searchBarTextDidBeginEditing(_ searchBar: UISearchBar) {
+        searchBar.tintColor = UIColor.black
+        searchBar.showsCancelButton = true
+    }
+   
+    
+    func searchBarCancelButtonClicked(_ searchBar: UISearchBar) {
```

Comment:
What do you think about moving `UISearchBarDelegate` methods to an extension of `CharactersViewController` to keep this file more organized?

```swift
extension CharactersViewController: UISearchBarDelegate {
    public func searchBarTextDidBeginEditing(_ searchBar: UISearchBar) {...}
    ...
    }
```